### PR TITLE
Enhancement: Extract ViewAction from TalksController

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] extracts a `Reviewer\Talk\ViewAction` from the `Reviewer\TalksController`

Follows #912.

### CODE REVIEW

File: classes/Http/Action/Reviewer/Talk/ViewAction.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        $this->talkHandler->grabTalk((int) $request->get('id'));
+
+        if (!$this->talkHandler->view()) {
+            $request->getSession()->set('flash', [
+                'type'  => 'error',
+                'short' => 'Error',
+                'ext'   => 'Could not find requested talk',
+            ]);
+
+            $url = $this->urlGenerator->generate('admin_talks');
```
Comment (localheinz) [inline classes/Http/Action/Reviewer/Talk/ViewAction.php]: Note how there's a mistake here, as the route `admin_talks` should be `reviewer_talks`.

File: classes/Http/Controller/Reviewer/TalksController.php [hunk 2]
```diff
@@ -35,25 +35,6 @@ public function __construct(TalkHandler $talkHandler, Twig_Environment $twig, Ur
...
-        $this->talkHandler->grabTalk((int) $request->get('id'));
-
-        if (!$this->talkHandler->view()) {
-            $request->getSession()->set('flash', [
-                'type'  => 'error',
-                'short' => 'Error',
-                'ext'   => 'Could not find requested talk',
-            ]);
-
-            return $this->redirectTo('admin_talks');
```
Comment (localheinz) [inline classes/Http/Controller/Reviewer/TalksController.php]: Note how there's a mistake here, as the route `admin_talks` should be `reviewer_talks`.

File: classes/Http/Action/Reviewer/Talk/ViewAction.php [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        Talk\TalkHandler $talkHandler,
+        Twig_Environment $twig,
+        Routing\Generator\UrlGeneratorInterface $urlGenerator
+    ) {
+        $this->talkHandler  = $talkHandler;
+        $this->twig         = $twig;
+        $this->urlGenerator = $urlGenerator;
+    }
+
+    public function __invoke(HttpFoundation\Request $request): HttpFoundation\Response
```
Comment (BackEndTea) [inline classes/Http/Action/Reviewer/Talk/ViewAction.php]: Is there a reason for not using ```@Template``` here?

Comment (localheinz) [reply classes/Http/Action/Reviewer/Talk/ViewAction.php]: Yes, the reason is that we would then have to remove the type declaration, as 

* for a redirect, we would need to return an instance of `HttpFoundation\Response`
* for the success case, we would need to return an array

**Comment by localheinz** [comment]
Thank you, @BackEndTea!

**Pull request merged**